spec/domain/command_handlers/game_command_handler_spec.rb

```diff
@@ -116,231 +137,335 @@
+        given_events(
+          GameCreatedEvent.new(id, GameState::LOBBYING, bob),
+          PlayerJoinedGameEvent.new(id, bob),
+          PlayerJoinedGameEvent.new(id, steven),
+          GameStartedEvent.new(id, GameState::RUNNING, deck, board),
+          SpawnPlacedEvent.new(id, bob, GameUnit.new(2, 1, GameUnit::DOWN)),
+          SpawnPlacedEvent.new(id, steven, GameUnit.new(3, 1, GameUnit::DOWN)),
+          GoalPlacedEvent.new(id, Goal.new(2, 11, 1)),
+          RobotSpawnedEvent.new(id, bob, GameUnit.new(2, 1, GameUnit::DOWN)),
+          RobotSpawnedEvent.new(id, steven, GameUnit.new(3, 1, GameUnit::DOWN)),
+          GameRoundStartedEvent.new(id, GameRound.new(1), {'bob' => [], 'steven' => []}, {'bob' => [], 'steven' => []}),
+          InstructionCardDealtEvent.new(id, bob, InstructionCard.u_turn(10)),
+          InstructionCardDealtEvent.new(id, steven, InstructionCard.u_turn(20)),
+          InstructionCardDealtEvent.new(id, bob, InstructionCard.u_turn(30)),
+          InstructionCardDealtEvent.new(id, steven, InstructionCard.u_turn(40)),
+          InstructionCardDealtEvent.new(id, bob, InstructionCard.u_turn(50)),
+          InstructionCardDealtEvent.new(id, steven, InstructionCard.u_turn(60)),
+          InstructionCardDealtEvent.new(id, bob, InstructionCard.rotate_left(70)),
+          InstructionCardDealtEvent.new(id, steven, InstructionCard.rotate_left(90)),
+          InstructionCardDealtEvent.new(id, bob, InstructionCard.rotate_left(110)),
+          InstructionCardDealtEvent.new(id, steven, InstructionCard.rotate_left(130)),
+          InstructionCardDealtEvent.new(id, bob, InstructionCard.rotate_left(150)),
+          InstructionCardDealtEvent.new(id, steven, InstructionCard.rotate_left(170)),
+          InstructionCardDealtEvent.new(id, bob, InstructionCard.rotate_left(190)),
+          InstructionCardDealtEvent.new(id, steven, InstructionCard.rotate_left(210)),
+          InstructionCardDealtEvent.new(id, bob, InstructionCard.rotate_left(230)),
+          InstructionCardDealtEvent.new(id, steven, InstructionCard.rotate_left(250)),
+          InstructionCardDealtEvent.new(id, bob, InstructionCard.rotate_left(270)),
+          InstructionCardDealtEvent.new(id, steven, InstructionCard.rotate_left(290)),
+          RobotProgrammedEvent.new(id, bob, [
+            InstructionCard.u_turn(10),
+            InstructionCard.u_turn(30),
+            InstructionCard.u_turn(50),
+            InstructionCard.rotate_left(70),
+            InstructionCard.rotate_left(110)
+          ]),
+          RobotProgrammedEvent.new(id, steven, [
+            InstructionCard.u_turn(20),
+            InstructionCard.u_turn(40),
+            InstructionCard.u_turn(60),
+            InstructionCard.rotate_left(90),
+            InstructionCard.rotate_left(130)
+          ]),
+          AllRobotsProgrammedEvent.new(id)
+        )
+      end
 
-        let(:tiles) do
-          BoardComposer.compose
-        end
+      it 'plays the round' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.